src/main/java/org/jabref/logic/importer/fetcher/CollectionOfComputerScienceBibliographiesParser.java

```diff
@@ -0,0 +1,101 @@
+package org.jabref.logic.importer.fetcher;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+import java.util.LinkedList;
+import java.util.List;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.jabref.logic.importer.ParseException;
+import org.jabref.logic.importer.Parser;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+public class CollectionOfComputerScienceBibliographiesParser implements Parser {
+    @Override
+    public List<BibEntry> parseEntries(InputStream inputStream) throws ParseException {
+        try {
+            Document document = buildDocumentFromInputStream(inputStream);
+            // uncomment to generate test case xml
+            // XMLUtil.printDocument(document);
+
+            NodeList childNodes = document.getChildNodes();
+            List<Element> itemElements = findItemElementsRecursively(childNodes);
+            List<BibEntry> bibEntries = parseItemElements(itemElements);
+
+            // uncomment to generate test case bib files
+            // System.out.println(bibEntries);
```

Comment:
Note that `toString()` in the BibTeX class **does not** produce "valid" BibTeX in all cases - see the method comment there. - However, in this case, it is fine.